frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -39,6 +39,27 @@ pods:
           server-properties:
             template: "{{CONFIG_TEMPLATE_PATH}}/server.properties.mustache"
             dest: "{{KAFKA_VERSION_PATH}}/config/server.properties"
+        readiness-check:
+          cmd: |
+            # since the server is just started, the starting and started log lines
```

Comment:
I would put this in a script file and include it as part of the distribution instead of putting the whole thing in the YAML.  It doesn't require any templating.